libcst/_parser/conversions/expression.py

```diff
@@ -1441,8 +1441,13 @@ def convert_arg_assign_comp_for(
         elt, for_in = children
         return Arg(value=GeneratorExp(elt.value, for_in, lpar=(), rpar=()))
     else:
-        # "key = value" assignment argument
         lhs, equal, rhs = children
+        # "key := value" assignment; positional
+        if equal.string == ":=":
+            val = convert_namedexpr_test(config, children)
+            assert isinstance(val, WithLeadingWhitespace)
```

Comment:
We don't use assert since it's noop in optimized mode. Let's just use if condition and raise when it returns unexpected result.
```suggestion
            if isinstance(val, WithLeadingWhitespace):
                return Arg(value=val.value)
            else:
                raise Exception("...")
```